R/00-setup_data.R

```diff
@@ -36,7 +36,12 @@ setup_data <- function(counts = NULL,
       pg_metadata = NULL,
```

Comment:
This was still NULL after I ran through setup, qc, and filtering steps. Unnecessary because the info that would normally be associated with it is handled in the annotate function?

Reply:
Yeah we could probably remove it. I don't think it ends up being used???